sources/packages/backend/libs/src-sql/sfas-restrictions/Bulk-insert-restrictions.sql

```diff
@@ -0,0 +1,93 @@
+/* 
+ * After all the SFAS restrictions are imported, 
+ * the below query inserts one Legacy restriction entry 
+ * per student if they have one or more restrictions 
+ * imported from the sfas system that are not present 
+ * in the sims restrictions.
+ */
+INSERT INTO
+  sims.student_restrictions (student_id, restriction_id)
+SELECT
+  DISTINCT(i.student_id),
+  (
+    SELECT
+      res.id
+    FROM
+      sims.restrictions res
+    WHERE
+      res.restriction_code = 'LGCY'
+  ) AS restriction_id
+FROM
+  (
+    SELECT
+      res.code,
+      CASE
+        WHEN res.code = 'A12' THEN '12'
+        WHEN res.code = 'AF1' THEN 'AF'
+        WHEN res.code = 'B2D' THEN 'B2'
+        ELSE res.code
+      END AS mapped_code,
+      res.individual_id,
+      res.is_included
+    FROM
+      sims.sfas_restrictions res
+    WHERE
+      res.removal_date IS NULL
+      AND res.is_included IS false
```

Comment:
Is there a need to use `is` operator here? 
`AND res.is_included = false` would not work for some reason?

Reply:
Both work. Changed it to equality.